samples/tokenCredentialSamples/AuthenticationCodeFlow/src/index.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+/**
+ * This sample is referenced from - https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/authorizationCodeSample.ts
+ */
+import { AuthorizationCodeCredential } from "@azure/identity";
+import express from "express";
+import { Server } from "http";
+import "isomorphic-fetch";
+import open from "open";
+import qs from "qs";
+
+import { TokenCredentialAuthenticationProvider } from "../../../../authProviders/azureTokenCredentials";
+import { Client } from "../../../../lib/src";
+import { port, tenantId, clientSecret, clientId, scopes, authorityHost, redirectUri } from "./secrets";
+
+if (tenantId === undefined || clientId === undefined) {
```

Comment:
is this all?  i thought you'd need a clientId and scopes as well.

Reply:
can you pass in common for the tenantid?

Reply:
This is a sample from  
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/manual/authorizationCodeSample.ts

Can you please explain what does common for the tenant id mean?

Reply:
>  @param tenantId - The Azure Active Directory tenant (directory) ID or name.
                   'common' may be used when dealing with multi-tenant scenarios.

https://github.com/Azure/azure-sdk-for-js/blob/9651b096b80d7e437a92266d2f5d16f75c9eb03a/sdk/identity/identity/src/credentials/authorizationCodeCredential.ts#L74